### DISCUSSION TITLE
[Jan 03] Doubt - Running noco using docker-compose

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1188] [Discord Threads]
> **DynamicDoyle** _queried at_ ```2022-01-02T16:12:39.530000+00:00```  
## I followed your Youtube video of how to deploy nocodb with docker-compose. This is the first time I really used Docker let alone Docker Compose. However, I can't figure out whether pg_nocodb_1 should be started or not. It won't let me. And when I try to create a postgres db following your Youtube video I'm not successful. Can you please help?  
  
> **o1lab** _commented at_ ```2022-01-02T16:28:55.323000+00:00```  
#### I just tried to run our docker-compose - appears to be good  
  
> **o1lab** _commented at_ ```2022-01-02T16:29:07.367000+00:00```  
#### can you see any errors when you run docker-compose in the logs  
  
> **o1lab** _commented at_ ```2022-01-02T16:30:17.317000+00:00```  
#### ```  
  
> **DynamicDoyle** _commented at_ ```2022-01-02T16:39:19.904000+00:00```  
#### A popup message says the port 8080 is already allocated  
  
> **DynamicDoyle** _commented at_ ```2022-01-02T16:40:53.882000+00:00```  
#### Since the nocodb is already running on 8080  
  
> **DynamicDoyle** _commented at_ ```2022-01-02T16:41:24.044000+00:00```  
#### shouldn't pg_nocodb_1 automatically use 5432 or some available port?  
  
> **o1lab** _commented at_ ```2022-01-02T16:42:17.575000+00:00```  
#### may be an issue docker gui  
  
> **o1lab** _commented at_ ```2022-01-02T16:42:54.349000+00:00```  
####   
  
> **o1lab** _commented at_ ```2022-01-02T16:42:56.764000+00:00```  
#### docker ps  
  
> **DynamicDoyle** _commented at_ ```2022-01-02T16:42:56.830000+00:00```  
####   
  
> **o1lab** _commented at_ ```2022-01-02T16:43:36.366000+00:00```  
#### pls do docker ps and see  
  
> **o1lab** _commented at_ ```2022-01-02T16:43:44.178000+00:00```  
#### did you run just nocodb before ?  
  
> **DynamicDoyle** _commented at_ ```2022-01-02T16:45:11.415000+00:00```  
####   
  
> **o1lab** _commented at_ ```2022-01-02T16:45:55.801000+00:00```  
#### netstat -ab  
  
> **DynamicDoyle** _commented at_ ```2022-01-02T16:48:50.599000+00:00```  
#### oh maybe it's wsl  
  
> **DynamicDoyle** _commented at_ ```2022-01-02T16:48:52.734000+00:00```  
####   
  
> **DynamicDoyle** _commented at_ ```2022-01-02T16:49:12.047000+00:00```  
####   
  
> **DynamicDoyle** _commented at_ ```2022-01-02T16:51:31.632000+00:00```  
#### or that could be the same application I guess  
  
> **DynamicDoyle** _commented at_ ```2022-01-02T16:59:00.424000+00:00```  
#### docker-compose up  
  
> **DynamicDoyle** _commented at_ ```2022-01-02T17:01:24.763000+00:00```  
#### if I stop the other 8080 port container then I can start the pg_nocodb_1  
  
> **DynamicDoyle** _commented at_ ```2022-01-02T17:06:41.414000+00:00```  
#### ok so if I remove elated_zhukovsky then I get a different error trying to create the db  
  
> **DynamicDoyle** _commented at_ ```2022-01-02T17:07:01.304000+00:00```  
####   
  
> **o1lab** _commented at_ ```2022-01-02T17:07:12.865000+00:00```  
#### you do not have to create to that db again  
  
> **o1lab** _commented at_ ```2022-01-02T17:07:20.223000+00:00```  
#### once you enter dashboard  
  
> **o1lab** _commented at_ ```2022-01-02T17:07:34.269000+00:00```  
#### you can use 'create project' option  
  
> **o1lab** _commented at_ ```2022-01-02T17:07:59.795000+00:00```  
#### https://docs.nocodb.com/#architecture---simple-overview  
  
> **DynamicDoyle** _commented at_ ```2022-01-02T17:09:27.706000+00:00```  
#### ah ok thank youu  
  
> **DynamicDoyle** _commented at_ ```2022-01-02T17:09:53.518000+00:00```  
#### it won't use sqlite then in this pg_nocodb_1  
  
> **o1lab** _commented at_ ```2022-01-02T17:10:11.216000+00:00```  
#### ya  
  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/927232937815064587)  

- - - -

#### Join [NocoDB's](http://www.nocodb.com/)  community
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)

**Comment by geek-black-li** [comment]
这是来自QQ邮箱的假期自动回复邮件。您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
